examples/sveltekit/flatbread.config.js

```diff
@@ -1,8 +1,8 @@
 import {
   defineConfig,
-  markdownTransformer,
-  yamlTransformer,
-  filesystem,
+  transformerMarkdown,
+  transformerYaml,
+  sourceFilesystem,
```

Comment:
[nit pick]

I like the pattern of using `sourceX` but for transformers I think it reads better if you have it as `xTransformer` instead of `transformerX`

Reply:
I'm fine with switching it to `xTransformer` but I think we should pick a consistent convention.

How do you feel about `filesystemSource`?

Reply:
hmm... I don't like that really either.  I think if we keep them consistent though the current change you made is probably the best since it starts with what it is too

trransformerX
sourceX

Reply:
what if we change the name of the import to be a noun like:

transformX

We could use that pattern with resolvers and providers too

resolveX
provideX

Reply:
markdownTransformer -> transformMarkdown
resolverSvimg -> resolveSvimg
providerApollo -> provideApollo
sourceFilesystem -> ?

I think I like that